schema/mysql/v57/cadence/database.sql

```diff
@@ -1 +1 @@
-CREATE DATABASE cadence;
\ No newline at end of file
+CREATE DATABASE cadence character set utf8;
```

Comment:
utf8mb4?

Reply:
utf8mb4 will limit the workflow_id length to 191 bytes instead of 255 bytes. On the other hand, the benefit you get with mb4 is support for additional unicode characters.. and it helps avoid some SQL injection attacks. We discussed this a bit last week and decided to go with utf8 because:
 - We don't need the support for additional unicode characters - we will update the documentation to make this clear
 - Cadence is not a customer facing service and probably will never be. So, the SQL injection is less of a concern, although its still legit